pom.xml

```diff
@@ -135,11 +135,13 @@
 				<artifactId>spring-cloud-deployer-local</artifactId>
 				<version>${spring-cloud-deployer-local.version}</version>
 			</dependency>
+<!--
```

Comment:
Could you add the reasoning behind removing the batch admin manager?

Reply:
Batch admin is not maintained anymore. Plain needed files copied over to get things working and revisit whole feature later. It's action point to batch guys.